travis/test_server.py

```diff
@@ -231,7 +233,6 @@ def setup_server(db, odoo_unittest, tested_addons, server_path, script_name,
     except subprocess.CalledProcessError:
         print("Using previous openerp_template database.")
     else:
-        # unbuffer keeps output colors
```

Comment:
Should we remove this comment?

Reply:
It's not removed, just moved to https://github.com/OCA/maintainer-quality-tools/pull/544/files#diff-4b42f344199e5694ffee3d2a57dd80c7R223